scripts/version

```diff
@@ -61,10 +61,6 @@ if echo $TAG | grep -q dirty; then
     TAG=dev
 fi
 
-if [ -n "$DRONE_TAG" ]; then
```

Comment:
Don't we need to keep this conditional (But with `GIT_TAG` instead of `DRONE_TAG`)? 

If I'm reading this correctly, `TAG` (and thus `AGENT_TAG`) will always be `${VERSION}${SUFFIX}`. Maybe I'm missing another area where `TAG` is set before this line?

Reply:
TAG is being set at the workflow level, because it is used in other places than the script. I'll add it back with the correct env names.